src/main/resources/templates/account_settings.html

```diff
@@ -21,6 +21,7 @@ <h2 class="team-heading team-heading-current-team" th:text="#{acc.info.header.ac
                             <label for="password"><span th:text="#{acc.info.password}">Password</span>:</label>
                             <input type="password" class="form-control" th:field="*{password}"/>
                             <span class="account-change-success" th:if="${editPassword eq 'success'}"><i class="fa fa-check"></i> <span th:text="#{acc.info.pwdSuccess}">TEXT</span></span>
+                            <span class="account-change-fail" th:if="${editPassword eq 'fail'}"><i class="fa fa-times"></i> <span th:text="#{acc.info.pwdFail}">TEXT</span></span>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)